generator_process/models/model_config.py

```diff
@@ -0,0 +1,60 @@
+import enum
+
+from ...absolute_path import absolute_path
+
+
+class ModelConfig(enum.Enum):
+    AUTO_DETECT = "auto-detect"
+    STABLE_DIFFUSION_1 = "v1"
+    STABLE_DIFFUSION_2_BASE = "v2 (512, epsilon)"
+    STABLE_DIFFUSION_2 = "v2 (768, v_prediction)"
+    STABLE_DIFFUSION_2_DEPTH = "v2 (depth)"
+    STABLE_DIFFUSION_2_INPAINTING = "v2 (inpainting)"
+    STABLE_DIFFUSION_XL_BASE = "XL (base)"
+    STABLE_DIFFUSION_XL_REFINER = "XL (refiner)"
+    CONTROL_NET_1_5 = "1.5 (ControlNet)"
+    CONTROL_NET_2_1 = "2.1 (ControlNet)"
```

Comment:
Do we need a separate case here for XL ControlNet models?

Reply:
Any XL ControlNet models I can find are only in diffusers format. I'm not aware of any yaml config files that can be used for converting or that I'm familiar enough with the parameters to mock one up.